web_dashboard_open_action/static/src/js/web_dashboard_open_action.js

```diff
@@ -19,17 +19,22 @@
 //
 //############################################################################
 
-openerp.web_dashboard_open_action = function(instance)
-{
-    instance.web.form.DashBoard.include({
+odoo.define('pos_all_orders_reprint.chrome_extensions', function(require) {
```

Comment:
Why the rename here? From what I understand, it should be `odoo.define('web_dashboard_open_action.bashboard_action'...)`